content/tutorials/grafana-fundamentals.md

```diff
@@ -75,27 +88,27 @@ The sample application, Grafana News, lets you post links and vote for the ones
 
 To add a link:
 
-- In the **Title** box, type "Example".
-- In the **URL** box, type "https://example.com".
+- In **Title**, enter **Example**.
+- In **URL**, enter **https://example.com**.
 - Click **Submit** to add the link.
 
-To vote for a link:
+The link appears in the list under the Grafana News heading.
 
-- In the list of links, click the triangle icon next to the name of the link.
+To vote for a link, click the triangle icon next to the name of the link.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Log in to Grafana" %}}
 
 Grafana is an open-source platform for monitoring and observability that lets you visualize and explore the state of your systems.
 
-- Browse to [localhost:3000](http://localhost:3000).
-- In the **email or username** box, type "admin".
-- In the **password** box, type "admin".
+- Browse to [localhost:3001](http://localhost:3001). ((MARCUS - We should probably note all cases where we change the default port to let them know why we did it and what the default port is.))
```

Comment:
Sorry, it should be the default port, i.e. 3000. This should be updated in the tutorial-environment repository.